etc/clean.py

```diff
@@ -1,18 +1,25 @@
+#!/usr/bin/env python
 import os
 
 
 def main():
     path = '.'
     count = 0
-    for name, folders, files in os.walk(path):
+    for name, folders, files in os.walk(path, topdown=False):
```

Comment:
what's the `topdown` arg for here?

Reply:
It is used so that `*.pyc` files would be deleted from `__pycache__` before we try to delete these directories using `os.rmdir()` (thus the dirs are preserved if they unexpected files).